docs/authentication/saml/azure.mdx

```diff
@@ -17,77 +17,83 @@ description: Learn how to allow users to sign into your Clerk app with their Mic
     }
   ]}
 >
-  - Use Microsoft Azure Entra ID (formerly [Active Directory](https://learn.microsoft.com/en-us/entra/fundamentals/new-name)) to enable single sign-on (SSO) via SAML for your Clerk application.
+  - Use Microsoft Azure Entra ID to enable SSO via SAML for your Clerk application.
 </TutorialHero>
 
-## Tutorial
+Enabling single sign-on (SSO) with **Microsoft Azure Entra ID** (formerly [Active Directory](https://learn.microsoft.com/en-us/entra/fundamentals/new-name)) allows your users to sign in and sign up to your Clerk application with their Microsoft account.
+
+## Configure for your production instance
+
+In _production instances_, you must provide custom credentials, which involves generating your own **Client ID** and **Client Secret** using your Microsoft Entra ID dashboard account.
+
+To make the setup process easier, it's recommended to keep two browser tabs open: one for your [**Clerk Dashboard**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [**Microsoft Azure portal**](https://portal.azure.com).
 
 <Steps>
   ### Set up an enterprise connection in Clerk
 
   To create a SAML connection in Clerk:
 
-  1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections).
+  1. Navigate to the [**Clerk Dashboard**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections).
```

Comment:
We don't ever bold Clerk Dashboard in the docs. If we were to start, we need to update that across the entire docs.

```suggestion
  1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections).
```